CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ All notable changes to this project will be documented in this file.
 - [`utils.compress_zstd`](docs/mod_utils.md#compress_zstd) compresses data using Facebook's Zstandard algorithm.
 - [`utils.decompress_zstd`](docs/mod_utils.md#decompress_zstd) decompresses data using Facebook's Zstandard algorithm.
 
+### Changed
+- **Debot Module**:
+    - [breaking] `fetch` function does't create an instance of debot. It returns DeBot metadata (`DeBotInfo`).
```

Comment:
Structure name is `DebotInfo`. Pay attention to names case

Reply:
FIxed